src/shadowbox/server/manager_service.ts

```diff
@@ -28,7 +28,7 @@ import {ServerConfigJson} from './server_config';
 import {SharedMetricsPublisher} from './shared_metrics';
 
 // Creates a AccessKey response.
-function accessKeyToJson(accessKey: AccessKey) {
+function accessKeyResponseJson(accessKey: AccessKey) {
```

Comment:
This rename goes along with the above rename of MakeAccessKeyJson to accessKeySerializedJson in order to better disambiguate the two different "Access Key to JSON" functions.